gemrb/core/Scriptable/Scriptable.cpp

```diff
@@ -2181,11 +2183,20 @@ void Movable::WalkTo(const Point &Des, int distance)
 		return;
 	}
 
+	Point smp_pos = Point(Pos.x/16, Pos.y/12);
+	Point smp_des = Point(Des.x/16, Des.y/12);
+	int hdist = int(Distance(smp_pos, smp_des) * 1.3);
```

Comment:
You can just use: `Distance(ConvertCoordToTile(Pos), ConvertCoordToTile(Des))`

Reply:
Done. I've also removed the tuple output in favor of the pointer param.